README.rst

```diff
@@ -639,6 +639,22 @@ Allows to specify a `list` of tuples for adding commands as described in
 
 Allows specifying JSONSchema used for validating meta-data of `Device Group <#device-groups>`_.
 
+``OPENWISP_CONTROLLER_SHARED_MANAGEMENT_IP_ADDRESS_SPACE``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``bool``  |
++--------------+-----------+
+| **default**: | ``False`` |
```

Comment:
I think we have to default this to True because most instances are deployed this way right now, instances which are used by multiple organizations having 1 management VPN for each org managed by the system is something I haven't heard  of yet, it may be possible that someone is doing it, but most users have only 1 management VPN for the entire system, so asking the majority of users to change this setting doesn't seems non very practical, what do you think?

This means that the old tests testing this feature may have to be changed, or you could mock the setting so it keeps working as before and those old tests can remain unchanged.

Reply:
I think you are right. I have updated the default value of this setting to True.